starter-project-backend-g31/src/helper/userprofilehelper.ts

```diff
@@ -1,5 +1,14 @@
-export function destructProfile(body: any){
-    var {_id,  name, username, bio, phone, avatar } = body
+import { AnyAaaaRecord } from 'dns'
+import cloudinary from '../utils/cloudinary'
+
+export async function destructProfile(body: any, fileRequest: any) {
+    var { _id, name, username, bio, phone } = body
+    var file = undefined
+    if (fileRequest) {
+            const result = await cloudinary.uploader.upload(fileRequest.path)
```

Comment:
uplade it to specic folder for Group 31, upload based on this.
cloudinary.uploader.upload(path, {
      use_filename: true,
      unique_filename: false,
      folder: configs.DOC_FOLDER_NAME
    })